packages/block-library/src/image/edit.js

```diff
@@ -425,10 +406,25 @@ class ImageEdit extends Component {
 				<Fragment>
 					{ controls }
 					<MediaPlaceholder
-						icon={ <BlockIcon icon={ icon } /> }
-						className={ className }
+						icon={ ! url ? <BlockIcon icon={ icon } /> : <img
+							alt={ __( 'Replace image' ) }
+							title={ __( 'Replace image' ) }
+							width={ '50%' }
+							className={ 'replace-image-preview' }
+							src={ url }
+						/> }
+						labels={ {
+							title: ! url ? __( 'Image' ) : <span
+								className={ 'replace-image-preview-title' }
+							>
+								{ __( 'Replace image' ) }
+							</span>,
+						} }
+						className={ classnames( className, ! url ? '' : 'wp-block-image-replace' ) }
```

Comment:
`classnames` supports a more canonical way of representing a class that is either "on" or "off":

```js
className={ classnames( className, {
	'wp-block-image-replace': url,
} ) }
```

Reply:
Cool!